packages/contracts-bedrock/deploy-config/plasma-devnetL1-template.json

```diff
@@ -0,0 +1,64 @@
+{
+  "l1ChainID": 900,
+  "l2ChainID": 901,
+  "l2BlockTime": 2,
+  "maxSequencerDrift": 300,
+  "sequencerWindowSize": 200,
+  "channelTimeout": 120,
```

Comment:
Do we need this entire extra config template? Its going to be painful to keep it up to date with the other template file, we should opt for having a single template file